ManagementPacks/Community.DataOnDemand/Scripts/Get-EventLogs.ps1

```diff
@@ -50,10 +50,18 @@ if ($EntryType) {
 
 $EventLogs = Get-EventLog @Params
 
+# Get properties of object to be displayed in output (Get-Memeber does not honor order of properties in object)
```

Comment:
The note is to make it clear if one needs to keep a specific ordering one should not use Get-Member. Though as the original code does not need to retain a custom object ordering (as default ordering of the event object is the same as Get-Member returns). Though this would not be true for every type of object. Therefore one need to check that the default object structure and what Get-Member returns is the same. Hence the comment.